### PR TITLE
Add Plugins page

### PR DESCRIPTION
**Pull request by jonataswalker**
This PR addresses #74.

### CODE REVIEW

**Comment by jonataswalker** [comment]
Ready for review, @tschaub.

File: src/pages/plugins/index.hbs [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+            <td>Layer control for OpenLayers 3.</td>
+            <td><a href="https://github.com/walkermatt">Matt Walker</a></td>
+          </tr>
+          <tr>
+            <td><a href="https://github.com/walkermatt/ol3-popup">OL3-Popup</a></td>
+            <td>Basic popup overlay for OpenLayers 3.</td>
+            <td><a href="https://github.com/walkermatt">Matt Walker</a></td>
+          </tr>
+          <tr>
+            <td><a href="https://github.com/bjornharrtell/jsts">JSTS</a></td>
```
Comment (marcjansen) [inline src/pages/plugins/index.hbs]: Is this a plugin for OL3?

Comment (jonataswalker) [inline src/pages/plugins/index.hbs]: I'm not sure how to call it, or should I remove it from this list?

Comment (marcjansen) [inline src/pages/plugins/index.hbs]: What do you suggest, @bjornharrtell?

Comment (bjornharrtell) [inline src/pages/plugins/index.hbs]: I don't think it fits in this listing.

Comment (ahocevar) [inline src/pages/plugins/index.hbs]: We could make this a list of libraries that integrate well with OpenLayers, not just plugins.

Comment (jonataswalker) [inline src/pages/plugins/index.hbs]: I like the idea @ahocevar and how could it be named (Add-ons?) in the top menu?

**Comment by jonataswalker** [comment]
We stopped at the point ... how this page will be named?

**Comment by ahocevar** [comment]
Good question. What about "Useful 3rd party libraries" as title and "3rd party" for the menu item?

**Comment by jonataswalker** [comment]
I'd stay with "Add-ons" but you're the boss.

**Comment by jonataswalker** [comment]
A little git messy, sorry. I'll soon fix it up.

File: src/pages/3rd-party/index.hbs [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+<div class="container">
+
+  <div class="row">
+    <h3 class="col-sm-12">Useful 3rd party libraries</h3>
+  </div>
+
+  <div class="row">
+    <div class="col-sm-12">
+      <p>
+        Here you can find some third-party libraries that can easily extend OpenLayers features.
```
Comment (ahocevar) [inline src/pages/3rd-party/index.hbs]: Maybe change to something like: "The libraries below extend or integrate well with OpenLayers."

**Comment by ahocevar** [comment]
Nevermind. I'm going to merge this and create a follow-up pull request. Thanks for getting this started @jonataswalker.

**Pull request merged**